src/java/fr/paris/lutece/plugins/appointment/web/AppointmentApp.java

```diff
@@ -870,6 +870,8 @@ public synchronized XPage doValidateForm( HttpServletRequest request ) throws Us
         AppointmentUtilities.fillAppointmentDTO( _notValidatedAppointment, nbBookedSeats, strEmail, strEmailConfirm, strFirstName, strLastName );
         AppointmentUtilities.validateFormAndEntries( _notValidatedAppointment, request, listFormErrors, false );
         AppointmentUtilities.fillInListResponseWithMapResponse( _notValidatedAppointment );
+        AppointmentUtilities.fillAppointmentPhoneNumber( _notValidatedAppointment, AppointmentUtilities.CONSTANT_GENERIC_ATTRIBUTE_TYPE_PHONE_NAME );
```

Comment:
this should be done in "fillAppointmentDto", before the call of "validateFormAndEntries"